.github/workflows/fmt.yaml

```diff
@@ -1,10 +1,10 @@
-# When pushed to main, run `cargo +nightly fmt` against all files and open a PR.
+# When pushed to master, run `cargo +nightly fmt` against all files and open a PR.
 name: rustfmt
 on:
   push:
-    # Limit to `main` because this action creates a PR
+    # Limit to `master` because this action creates a PR
```

Comment:
I'll rename the default branch shortly 😓 

It's been a long time since I gave any attention to these repos.

Reply:
It's renamed to `main` now.